app/mempool/lane.go

```diff
@@ -33,6 +33,12 @@ type Lane struct {
 	// currently in this lane's mempool.
 	txIndex map[string]struct{}
 
+	// handleLaneLimitCheck is a callback function that is called when the lane exceeds its limit.
+	handleLaneLimitCheck func(isLaneLimitExceeded bool)
```

Comment:
suggestion: callbackAfterFillProposal // I think it always call no matter limit reach or not?